src/tiffimage_int.cpp

```diff
@@ -749,7 +749,7 @@ namespace Exiv2 {
     //! Sony1 Camera Settings binary array - configuration
     extern const ArrayCfg sony1CsCfg = {
         sony1CsId,        // Group for the elements
-        bigEndian,        // Big endian
```

Comment:
Why the sudden whitespace changes now?

Reply:
What whitespace changes?  In the code, in the output?

Reply:
Whitespace changes were done by VS, regrettably. It changed the spaces to tabs. I have changed the setting that prevents this from happening again.